---
layout              : default
title               : "CV"
permalink           : "/cv/"
---
<h1 style="text-align: center;">Zachary L. Skidmore</h1>
<p style="text-align: center;">Saint Louis, Mo | ORCiD: <a href="https://orcid.org/0000-0003-1928-7139">0000-0003-1928-7139</a> | <a href="https://github.com/zlskidmore">github.com/zlskidmore</a></p>
---
#### Eductation:
<div class="container">
  <div class="row">
    <div class="col">
    University of Illinois
    </div>
    <div class="col" style="text-align: right;">
    Chicago, Illinois
    </div>
  </div>
  <div class="row">
    <div class="col">
    Master of Engineering - Bioinformatics
    </div>
    <div class="col" style="text-align: right;">
    May 2013
    </div>
  </div>
</div>
---
#### Work Experience:
<div class="container">
  <div class="row">
    <div class="col">
    McDonnell Genome Institute, Washington University Department of Medicine
    </div>
    <div class="col" style="text-align: right;">
    Saint Louis, Missouri
    </div>
  </div>
  <div class="row">
    <div class="col">
    Position: Staff Scientist
    </div>
    <div class="col" style="text-align: right;">
    November 2013 - Present
    </div>
  </div>
</div>

- 2020 - 2022 Highlights

    - Lead author and analyst for a study of non-cirrotic hepatocellular carcinoma (WGS/WES/Custom Capture) (Manuscript Submitted)
    - Lead author and analyst for a study of small cell lung cancer (WGS/WES/RNAseq) (Unpublished)
    - Lead author and analyst of a protocol for scTCRseq and scRNAseq for canine samples (Manuscript in progress)
    - Implemented modifications to existing H. sapiens workflows to be compatible with multiple species PMID: [33556121](https://pubmed.ncbi.nlm.nih.gov/33556121/)
    - Built a custom workflow to detect viral integration sites from dual indexed viral reads PMID: [34019588](https://pubmed.ncbi.nlm.nih.gov/34019588/)
    - Implemented machine learning algorithm to classify and refine somatic variant calls using extreme gradient boosting models
    - Optimized analysis scripts for the RegTools software package to be faster and less memory intensive.

- 2018 - 2019 Highlights

    - Created and instituted Common Workflow Language (CWL) bioinformatic pipelines: [https://github.com/griffithlab/bioinformatic-workflows](https://github.com/griffithlab/bioinformatic-workflows); [https://github.com/genome/analysis-workflows](https://github.com/genome/analysis-workflows)
    - Created and maintained docker images for bioinformatic analysis: [https://hub.docker.com/u/zlskidmore](https://hub.docker.com/u/zlskidmore)
    - Began Training Staff Scientists in various bioinformatic and programming skill sets.
    - Created static websites and subsequent material for genomic visualization, precision medicine, and RNAseq.
    - Advised and/or performed analysis on TCR clonotype repertoire, HLA prediction, neoantigen prediction, ctDNA, WGS, WES, RNAseq PMIDs: [31390566](https://pubmed.ncbi.nlm.nih.gov/31390566/), [31618044](https://pubmed.ncbi.nlm.nih.gov/31618044/), [31467059](https://pubmed.ncbi.nlm.nih.gov/31467059/), [31685621](https://pubmed.ncbi.nlm.nih.gov/31685621/), [31902496](https://pubmed.ncbi.nlm.nih.gov/31902496/), [32665297](https://pubmed.ncbi.nlm.nih.gov/32665297/), [34036230](https://pubmed.ncbi.nlm.nih.gov/34036230/), [33556121](https://pubmed.ncbi.nlm.nih.gov/33556121/), [33547198](https://pubmed.ncbi.nlm.nih.gov/33547198/)
    - Prototyped shiny application for interrogation of saturation mutagenesis assays: [griffithlab/saturation-mutagenesis-viz](https://github.com/griffithlab/saturation-mutagenesis-viz)

- 2016 - 2017 Highlights

    - Published Genomic Visualization in R (GenVisR) library on bioconductor [DOI: 10.18129/B9.bioc.GenVisR](https://doi.org/doi:10.18129/B9.bioc.GenVisR)
    - Consulted on design and implementation of Clinical Interpretations of Variants in Cancer (CIViC) database; Co-organized first Biohackathon for CIViC at the Netherlands Cancer Institute; Submitted and curated database entries PMID: [28138153](https://pubmed.ncbi.nlm.nih.gov/28138153/)
    - Began mentoring and assisting graduate and undergraduate students in analyzing projects and writing manuscripts.

- 2013 - 2015 Highlights

    - Assisted in the maintenance and improvement of institute project management and analysis software (perl) PMID: 26645048
    - Conceived and developed genomic visualization software (R) PMIDs: 27288499, 34506690
    - Conducted statistical and bioinformatic analysis for next generation sequencing data (Benchmarking Alignment, Clonotype Expansion, Cancer Genomic Alterations, etc.): PMID [26531824](https://pubmed.ncbi.nlm.nih.gov/26531824/), [26563128](https://pubmed.ncbi.nlm.nih.gov/26563128/), [27058228](https://pubmed.ncbi.nlm.nih.gov/27058228/), [27029710](https://pubmed.ncbi.nlm.nih.gov/27029710/), [27181063](https://pubmed.ncbi.nlm.nih.gov/27181063/), [27681435](https://pubmed.ncbi.nlm.nih.gov/27681435/)
    - Performed next generation sequencing analysis and presented results for cancer genomic tumor board n-of-1 cases

---
#### Teaching Experience:
- Instructor/Contributing Author for genomic visualizations workshop ([https://genviz.org](https://genviz.org))
    - [Physalia-courses](https://www.physalia-courses.org/), Freie Universität Berlin, Germany
    - [Evomics](http://evomics.org/) - Workshop on Genomics 2019, Český Krumlov, Czech Republic
- Teaching assistant/Contributing Author for Precision Medicine workshop ([https://pmbio.org](https://pmbio.org))
    - [PR Informatics](https://www.prinformatics.com/), Glasgow, Scotland
- Contributing Author for RNAseq workshop ([https://rnabio.org](https://rnabio.org))

---
#### Developer Skills (Abbreviated):
<table class="table">
  <thead class="thead-dark">
    <tr>
      <th scope="col">Advanced</th>
      <th scope="col"></th>
      <th scope="col">Profecient</th>
      <th scope="col">Familiar</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>- R (inc. ggplot2, data.table, S4, etc.)</td>
      <td>- LSF</td>
      <td>- Amazon Web Services (AWS)</td>
      <td>- SQL</td>
    </tr>
    <tr>
      <td>- UNIX</td>
      <td>- Confluence/JIRA</td>
      <td>- CWL</td>
      <td>- Python</td>
    </tr>
    <tr>
      <td>- GNU Utils (incl. Awk, sed, vim, etc.)</td>
      <td></td>
      <td>- Perl</td>
      <td>- CSS/SASS</td>
    </tr>
    <tr>
      <td>- Docker</td>
      <td></td>
      <td>- Jekyll</td>
      <td></td>
    </tr>
    <tr>
      <td>- Git/GitHub</td>
      <td></td>
      <td>- HTML5</td>
      <td></td>
    </tr>
  </tbody>
</table>

#### Analysis Skills (Abbreviated):
<table class="table">
  <thead class="thead-dark">
    <tr>
      <th scope="col">Advanced</th>
      <th scope="col"></th>
      <th scope="col">Profecient</th>
      <th scope="col"></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>- Next Generation Sequencing Analysis</td>
      <td>- Cancer Genomics</td>
      <td>- Statistical Analysis</td>
      <td>- Cancer Immunotherapy</td>
    </tr>
    <tr>
      <td>- Genomic Visualization</td>
      <td>- Data Mining</td>
      <td>- Machine Learning</td>
      <td>- TCRseq</td>
    </tr>
    <tr>
      <td>- RNAseq</td>
      <td>- Germline Analysis</td>
      <td>- scRNAseq</td>
      <td>- Experimental Design</td>
    </tr>
  </tbody>
</table>
